configen/FileGenerator.swift

```diff
@@ -131,6 +137,18 @@ struct FileGenerator {
     return line
   }
 
+  private func formatArrayString(rawValue: AnyObject, rawType: String) -> String {
+    var rawTypeCopy = rawType
+    var rawValueStr = "\(rawValue)"
+
+    rawTypeCopy = String(rawTypeCopy.dropFirst()) // drop [
+    rawTypeCopy = String(rawTypeCopy.dropLast()) // drop ]
+
+    rawValueStr = String(rawValueStr.dropFirst()) // drop (
+    rawValueStr = String(rawValueStr.dropLast()) // drop )
+
+    return "Array<\(rawTypeCopy)>(arrayLiteral: \(rawValueStr))"
```

Comment:
How will this output in the generated file, will it literally be `Array<String>(arrayLiteral: ...)`? Or will it be the nicer `[...]` style?

Reply:
Yeah, fixed now! 👍